docs/templates/parameters.md

```diff
@@ -247,39 +291,55 @@ data "coder_parameter" "cpu" {
-   - Mark the new parameter as `mutable`, so that Coder will not block updating existing workspaces.
+   - Add `coder_parameter` resource matching the legacy variable to
+     migrate.
+   - Use `legacy_variable_name` and `legacy_variable` to link the
+     `coder_parameter` to the legacy variable.
+   - Mark the new parameter as `mutable`, so that Coder will not block
+     updating existing workspaces.
 
-2. Update all workspaces to the updated template version. Coder will populate the added `coder_parameter`s with values from legacy variables.
+2. Update all workspaces to the updated template version. Coder will
+   populate the added `coder_parameter`s with values from legacy
+   variables.
 3. Prepare another template version:
 
    - Remove the migrated variables.
-   - Remove properties `legacy_variable` and `legacy_variable_name` from `coder_parameter`s.
+   - Remove properties `legacy_variable` and `legacy_variable_name`
+     from `coder_parameter`s.
 
 4. Update all workspaces to the updated template version (2nd).
 5. Prepare a third template version:
 
-   - Enable the `feature_use_managed_variables` provider flag to use managed Terraform variables for template customization. Once the flag is enabled, legacy variables won't be used.
+   - Enable the `feature_use_managed_variables` provider flag to use
+     managed Terraform variables for template customization. Once the
+     flag is enabled, legacy variables won't be used.
 
 6. Update all workspaces to the updated template version (3rd).
 7. Delete legacy parameters.
 
-As a template improvement, the template author can consider making some of the new `coder_parameter` resources `mutable`.
+As a template improvement, the template author can consider making
+some of the new `coder_parameter` resources `mutable`.
 
 ## Terraform template-wide variables
 
-> ⚠️ Flag `feature_use_managed_variables` is available until v0.25.0 (Jul 2023) release. After this release, template-wide Terraform variables will be enabled by default.
+> ⚠️ Flag `feature_use_managed_variables` is available until v0.25.0
+> (Jul 2023) release. After this release, template-wide Terraform
+> variables will be enabled by default.
 
-As parameters are intended to be used only for workspace customization purposes, Terraform variables can be freely managed by the template author to build templates. Workspace users are not able to modify
+As parameters are intended to be used only for workspace customization
+purposes, Terraform variables can be freely managed by the template
+author to build templates. Workspace users are not able to modify
 template variables.
 
-The template author can enable Terraform template-wide variables mode by specifying the following flag:
+The template author can enable Terraform template-wide variables mode
+by specifying the following flag:
 
 ```hcl
 provider "coder" {
   feature_use_managed_variables = "true"
 }
```

Comment:
<i class='graphite__hidden'>[Re: lines 339 to 339]<br/></i>

```suggestion
```<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/coder/coder/9139?utm_source=unchanged-line-comment">Graphite</a>.</span>